__tests__/environment-test.js

```diff
@@ -0,0 +1,81 @@
+"use strict"
+
+jest.dontMock('../environment.js');
+
+describe('environment', () => {
+  let environment = require('../environment.js');
+
+  it('uses original config attributes if no env key is found', () => {
+    let parsedConfig = environment.checkEnvironmentForConfig({
+      key:'fromDefaultConfig'
+    })
+
+    expect(parsedConfig).toEqual({
+      key:'fromDefaultConfig'
+    });
+  })
+
+  it('correctly interperts the current config', () => {
+    process.env.MENTION_BOT_CONFIG = `
```

Comment:
you want to cleanup that global to what it was before otherwise it's going to mess up subsequent tests